work/v1/types.go

```diff
@@ -94,6 +94,8 @@ type ManifestConfigOption struct {
 
 	// FeedbackRules defines what resource status field should be returned. If it is not set or empty,
 	// no feedback rules will be honored.
+	// +listType:=map
+	// +listMapKey:=type
```

Comment:
Here, I am not sure if we should treat the `type` as the map key, if we do, If we do this, it may be incompatible, but if we do not, users can still configure two same name jsonpaths:
```
feedbackRules:
- type: JSONPaths
   jsonPaths:
   - name: "Replica"
      path: .spec.replicas
- type: JSONPaths
   jsonPaths:
   - name: "Replica"
      path: .status.replicas
```

@qiujian16 @zhiweiyin318 WDYT?

Reply:
let's do name at first...I am not sure if anyone uses multiple jsonPaths today, not quite likely.